src/components/Patient/PatientHome.tsx

```
@@ -376,10 +376,17 @@ export const PatientHome = (props: {
                             .treating_physician_object,
                         )}
                       </p>
-                      <p className="flex items-end text-xs font-normal leading-tight">
+                      <span className="tooltip text-xs text-secondary-800 flex items-end  font-normal leading-tight">
                         {!!skillsQuery.data?.results?.length &&
                           formatSkills(skillsQuery.data?.results)}
-                      </p>
+                        {(skillsQuery.data?.results?.length || 0) > 3 && (
+                          <ul className="tooltip-text tooltip-bottom flex flex-col text-xs font-medium">
+                            {skillsQuery.data?.results.map((skill) => (
+                              <li>{skill.skill_object.name}</li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)